torch/_torch_docs.py

```diff
@@ -4625,6 +4624,53 @@ def merge_dicts(*dicts):
     torch.return_types.median(values=tensor([-0.3982,  0.2270,  0.2488,  0.4742]), indices=tensor([1, 4, 4, 3]))
 """.format(**single_dim_common))
 
+add_docstr(torch.nanmedian,
+           r"""
+nanmedian(input) -> Tensor
+
+This is a variant of :func:`torch.median` that "ignores" ``NaN`` values,
+computing the median as if ``NaN`` values in :attr:`input` did not exist.
```

Comment:
Open with the one sentence description: "Computes the median of :attr:\`input\`, ignoring \`\`NaN\`\`s.

Then you have your current paragraph, which is good and which elaborates on the function's behavior, but needs a couple tweaks:

"This function is identical to :func:\`torch.median\` when there are no ``NaN`` values in :attr:\`input\`. When :attr:\`input\` has one or more ``NaN`` values, :func:\`torch.median\` will always return ``NaN``, while this function will return the median of the non-``NaN`` elements of :attr:\`input\`. If all the elements of :attr:\`input\` are ``NaN`` it will also return ``NaN``."